### DISCUSSION TITLE
Is there a way to send custom metrics to elastic APM in python?

### DISCUSSION THREAD

**Discussion by Saketsuraj** [prometheus/client_python#801] [Q&A]
I cannot find a code example to send custom metrics to elastic. Please provide a code example in python.

**Comment by csmarchbanks** [comment]
I would recommend looking into if Elastic is able to collect metrics from /metrics endpoints or Prometheus remote write. This is a Prometheus library and documentation will be focused on interacting with the Prometheus ecosystem, not Elastic.

**Reply by Saketsuraj** [reply]
Actually, it is emitting custom metrics and getting collected in elastic using Prometheus metrics integration by starting the server at port 9090. Might help someone else as well.
